content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,281 @@
+and create a context for each cluster. Call the contexts `context-1`, `context-2`, and `context-3` corresponding to the 3 clusters.
+Finally, set the `KUBECONFIG` environment variable to the path of the configuration file you just created.
+
+## Preparing common state
+
+You first need to prepare a few files which you'll need during the subsequent steps.
+
+1. Create a working directory to hold all the files you'll be creating.
+
+    {{< text bash >}}
+    $ mkdir mc
+    $ cd mc
+    {{< /text >}}
+    
+1. Create a base Istio deployment configuration for all clusters in the mesh. This starts with the
+default profile, disables self-signed certs, and enables mutual TLS by default. You’ll be creating an external root and intermediate certs and
+plugging them into Citadel later.
+
+    {{< text bash >}}
+    $ cat > base.yaml << EOF
+    apiVersion: install.istio.io/v1alpha2
+    kind: IstioControlPlane
+    spec:
+      values:
+        security:
+          selfSigned: false
+        global:
+          mtls:
+            enabled: true
+    EOF
+    {{< /text >}}
+
+1. Now, create the `prepcluster.sh` script. It'll be used to create the CA certificates and keys in all the clusters,
+and create common service accounts.
+
+    {{< text bash >}}
+    $ cat > prepca.sh << EOF
+    $ #!/bin/bash
+    $ pushd mesh-certs
+    $ make intermediate-$(kubectl config current-context)-certs
+    $ cd intermediate-$(kubectl config current-context)
+    $ kubectl create namespace istio-system >/dev/null || true
+    $ kubectl create secret generic cacerts -n istio-system \
+            --from-file=ca-cert.pem \
+            --from-file=ca-key.pem \
+            --from-file=root-cert.pem \
+            --from-file=cert-chain.pem \
+            --dry-run -o yaml| kubectl apply -f -
+    $ popd
+    $ kubectl create clusterrole istio-reader-istio-system --verb=get,list,watch \
+        --resource=nodes,pods,endpoints,services,replicasets.apps,replicationcontrollers --dry-run -o yaml \
+        | kubectl apply -f -
+     
+    $ kubectl create serviceaccount istio-reader-service-account -n istio-system  --dry-run -o yaml \
+        | kubectl apply -f -
+     
+    $ kubectl create clusterrolebinding istio-reader-istio-system --clusterrole=istio-reader-istio-system \
+         --serviceaccount=istio-system:istio-reader-service-account  --dry-run -o yaml \
+        | kubectl apply -f -
+    EOF
+    $ chmod +x prepca.sh
+    $ istioctl x multicluster generate -f mesh-topology.yaml --from base.yaml > istio-$(kubectl config current-context).yaml
+    $ istioctl manifest apply -f istio-$(kubectl config current-context).yaml
+    $ for CONTEXT in $(kubectl config get-context -o name); do
+        $ istioctl --context ${CONTEXT} x multicluster generate --from base.yaml -f mesh-topology.yaml > istio-${CONTEXT}.yaml
+        $ istioctl --context ${CONTEXT} manifest apply -f istio-${CONTEXT}.yaml
+    $ done
+    {{< /text >}}
```

Comment:
These lines don't belong. Everything but the first manifest generation need to be run after the previous cluster was setup. Existing clusters need to be updated with the new cluster's ingress gateway allocated external IP. The manifest itself should be applied after the cluster is prepared (intermediate CA and service account).